streams-runtimes/streams-runtime-threaded/src/main/java/org/apache/streams/threaded/tasks/StatusCounts.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.streams.threaded.tasks;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import org.apache.streams.jackson.StreamsJacksonMapper;
+
+import java.io.Serializable;
+
+public class StatusCounts implements Serializable {
+
+    private String id;
+    private String type;
+    private long working;
+    private long success;
+    private long failed;
+    private long timeSpentSuccess;
+    private long timeSpentFailure;
+
+    public String getId()                   { return id; }
```

Comment:
Please check the formatting in this class as it doesn't match the conventions in the rest of the project.